notebooks/advanced_algorithms/Quantum_Computing_Quantum_Monte_Carlo.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
<p>Nit: local simulator usually refers to the Braket local simulator. Might be worth specifying local lightning simulator</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-braket-algorithm-library/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>